packages/integrations/preact/src/client-dev.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
Q: Was the `preact/debug` import erroring? If we're lacking type definitions we can easily PR that to `preact`.

Reply:
Yep, `preact/debug` doesn't have types at the moment: https://unpkg.com/browse/preact@10.11.0/debug/

Reply:
Oh no, filed a PR. Next version will have typings for that: https://github.com/preactjs/preact/pull/3732